src/System.Net.WebClient/tests/WebClientTest.cs

```diff
@@ -18,7 +18,7 @@ namespace System.Net.Tests
     public class WebClientTest
     {
         [Fact]
-        public static void DefaultCtor_PropertiesReturnExpectedValues()
+        public static void WebClient_DefaultCtor_PropertiesReturnExpectedValues()
```

Comment:
I remember that some of the CI reports were not including the class name. If that not true, I can revert this change.